Content.Server/Explosion/EntitySystems/ExplosionSystem.Processing.cs

```diff
@@ -181,27 +181,27 @@ public bool IsBlockingTurf(EntityUid uid, EntityQuery<PhysicsComponent> physicsQ
         // enumerator-changed-while-enumerating errors.
         List<(EntityUid, TransformComponent?) > list = new();
 
-        EntityUidQueryCallback callback = uid =>
+        void AddIntersecting(List<(EntityUid, TransformComponent?)> list)
         {
-            if (processed.Contains(uid))
-                return;
+            foreach (var uid in _entityLookup.GetLocalEntitiesIntersecting(lookup, ref gridBox, LookupFlags.None))
+            {
+                if (processed.Contains(uid))
+                    continue;
 
-            if (!xformQuery.TryGetComponent(uid, out var xform))
-                return;
+                if (!xformQuery.TryGetComponent(uid, out var xform))
+                    continue;
 
-            if (xform.ParentUid != grid.GridEntityId)
-            {
-                if (!metaQuery.TryGetComponent(uid, out var meta))
-                    return;
-                // Not parented to grid. Likely in a container.
-                if (_containerSystem.IsEntityInContainer(uid, meta))
-                    return;
-            }
+                if (xform.ParentUid != grid.GridEntityId)
+                {
+                    if (!metaQuery.TryGetComponent(uid, out var meta))
+                        continue;
+                }
```

Comment:
Getting the metadata via the metaQuery was only used for the container checks, so this can just be removed now.